cmd/dockerregistry/main.go

```diff
@@ -5,7 +5,6 @@ import (
 	"fmt"
 	"math/rand"
 	"net/http"
-	_ "net/http/pprof"
```

Comment:
This will break the `startProfiler` function.

Reply:
though i don't know that we've had a use for it so far, i'd be reluctant to ship code that we can't profile at a customer site if need be.  is there a way to make this configurable?

Reply:
It seems to me that I haven't done the due diligence on this one. Let me take a look.